src/riak_core_throttle.erl

```diff
@@ -110,6 +156,38 @@ get_limits(Key) ->
 clear_limits(Key) ->
     application:unset_env(riak_core, ?THROTTLE_LIMITS_KEY(Key)).
 
+%% @doc Returns a fun that used in Cuttlefish translations to translate from
+%% configuration items of the form:
+%%  `ConfigPrefix'.throttle.tier1.`LoadFactorMeasure'
+%% to the list of tuples form expected by the `set_limits/2' function in this
+%% module. See riak_kv.schema and yokozuna.schema for example usages.
+create_limits_translator_fun(ConfigPrefix, LoadFactorMeasure) ->
+    fun(Conf) ->
+            %% Grab all of the possible names of tiers so we can ensure that
+            %% both mailbox_size and delay are included for each tier.
```

Comment:
Should change the comment to no longer include `mailbox_size` as that's not always the name... should be `LoadFactorMeasure` now IIRC.